### ISSUE TITLE
hidden files and users/superusers

### ISSUE THREAD

**Issue by musicnerd** [ELVIS-Project/ELVIS-Database-Content#1]
Does the adding of the copyright clause allow us to put back up material that was previously "hidden"?  For example, to a non-superuser a search for "Bach, J.S." produces 1,118 results whereas for a "regular" user the result is only 7. Most users will NOT be superusers so it behooves us to attempt to put back as much "hidden" material as possible.

**Comment by fujinaga** [comment]
Yes, and the searches should return all 1,118 results whether they are logged in or not.
They can download them only if there are logged in, however.
